posthog/api/capture.py

```diff
@@ -50,6 +50,10 @@ def parse_kafka_event_data(
     token: str,
 ) -> Dict:
     logger.debug("parse_kafka_event_data", token=token, team_id=team_id)
+    properties = json.dumps(data["properties"])
```

Comment:
Is it possible that "properties" isn't in the data dict? Will we raise a 400?

Reply:
it shouldn't be - although indeed looking at it in isolation like this does seem to show that. I can add some handling here but we do earlier on set it to `{}` if it's empty. Tests actually verify this.